Modules/Microsoft365DSC/DSCResources/MSFT_IntuneVPNConfigurationPolicyIOS/MSFT_IntuneVPNConfigurationPolicyIOS.psm1

```diff
@@ -0,0 +1,1333 @@
+        [Microsoft.Management.Infrastructure.CimInstance[]]
+        $Assignments,
+        #endregion
+
+        [Parameter()]
+        [System.String]
+        [ValidateSet('Absent', 'Present')]
+        $Ensure = 'Present',
+
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $ApplicationSecret,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint,
+
+        [Parameter()]
+        [Switch]
+        $ManagedIdentity,
+
+        [Parameter()]
+        [System.String[]]
+        $AccessTokens,
+
+        #latest updates
+        [Parameter()]
+        [System.UInt32]
+        $version,
+
+        [Parameter()]
+        [System.String]
+        $loginGroupOrDomain,
+
+        [Parameter()]
+        [System.String]
+        $role,
+
+        [Parameter()]
+        [System.String]
+        $realm,
+
+        [Parameter()]
+        [System.String]
+        $identifier,
+
+        [Parameter()]
+        [System.Boolean]
+        $enablePerApp,
+
+        [Parameter()]
+        [ValidateSet('notConfigured', 'appProxy', 'packetTunnel')]
+        [System.String]
+        $providerType,
+
+        [Parameter()]
+        [System.Boolean]
+        $disableOnDemandUserOverride,
+
+        [Parameter()]
+        [System.Boolean]
+        $disconnectOnIdle,
+
+        [Parameter()]
+        [System.UInt32]
+        $disconnectOnIdleTimerInSeconds,
+
+        [Parameter()]
+        [System.String]
+        $microsoftTunnelSiteId,
+
+        [Parameter()]
+        [System.String]
+        $cloudName,
+
+        [Parameter()]
+        [System.Boolean]
+        $strictEnforcement,
+
+        [Parameter()]
+        [System.String]
+        $userDomain
+    )
+
+    try
+    {
+        $ConnectionMode = New-M365DSCConnection -Workload 'MicrosoftGraph' `
+            -InboundParameters $PSBoundParameters
+    }
+    catch
+    {
+        Write-Verbose -Message 'Connection to the workload failed.'
+    }
+
+    #Ensure the proper dependencies are installed in the current environment.
+    Confirm-M365DSCDependencies
+
+    #region Telemetry
+    $ResourceName = $MyInvocation.MyCommand.ModuleName.Replace('MSFT_', '')
+    $CommandName = $MyInvocation.MyCommand
+    $data = Format-M365DSCTelemetryParameters -ResourceName $ResourceName `
+        -CommandName $CommandName `
+        -Parameters $PSBoundParameters
+    Add-M365DSCTelemetryEvent -Data $data
+    #endregion    
+
+    $nullResult = $PSBoundParameters
+    $nullResult.Ensure = 'Absent'
+    try
+    {
+        if (-not [string]::IsNullOrWhiteSpace($id))
+        { 
+        $getValue = Get-MgBetaDeviceManagementDeviceConfiguration -DeviceConfigurationId $id -ErrorAction SilentlyContinue 
```

Comment:
Missing spaces

Reply:
Not fully sure what you mean. Have I tabbed instead of using spaces? I just re-indented with spaces and also pushed $getvalue 4 spaces inwards?

Reply:
It's currently aligned to the bracket, it needs to be indented 4 spaces inwards.

Reply:
Cool, that's what I did!